cargo-insta/tests/main.rs

```diff
@@ -81,6 +81,17 @@ fn assert_success(output: &std::process::Output) {
     );
 }
 
+fn assert_failure(output: &std::process::Output) {
+    eprint!("{}", String::from_utf8_lossy(&output.stderr));
+    eprint!("{}", String::from_utf8_lossy(&output.stdout));
+    assert!(
+        !output.status.success(),
+        "Tests succeeded: {}\n{}",
```

Comment:
```suggestion
        "Tests failed as expected: {}\n{}",
```

Reply:
This is in the case that the assert doesn't hold. But this message is indeed confusing, so I've changed it to `"Tests unexpectedly succeeded"`.

Reply:
Sorry yes, thanks